### PR TITLE
Show pypy2 in example

### PR DESCRIPTION
**Pull request by hugovk**
Make it clear to use `pypy2` not plain `pypy` for the Python 2 version, see:

* https://github.com/actions/setup-python/issues/15#issuecomment-525610040

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ jobs:
     runs-on: ubuntu-16.04
     strategy:
       matrix:
-        python: [ '2.x', '3.x', 'pypy3' ]
+        python: [ '2.x', '3.x', , 'pypy2', 'pypy3' ]
```
Comment (radarhere) [inline README.md]:
```suggestion
        python: [ '2.x', '3.x', 'pypy2', 'pypy3' ]
```

Comment (hugovk) [reply README.md]: Thanks!

**Comment by konradpabjan** [comment]
Hey @hugovk thanks for the PR and apologies for the huge wait. If you resolve the merge conflicts then we can get this in 😃

**Comment by hugovk** [comment]
Done, thanks!

**Pull request merged**